Library/Homebrew/requirements/minimum_macos_requirement.rb

```diff
@@ -3,18 +3,32 @@
 class MinimumMacOSRequirement < Requirement
   fatal true
 
-  def initialize(tags)
-    @version = MacOS::Version.from_symbol(tags.first)
+  def initialize(tags = [])
+    @version = MacOS::Version.from_symbol tags.first unless tags.empty?
     super
   end
 
-  satisfy(build_env: false) { MacOS.version >= @version }
+  satisfy(build_env: false) do
+    if @version
```

Comment:
`next false if !OS.mac? || !@version`

Reply:
May be worth taking the same approach below, too.

Reply:
Done. Thanks for the suggestion, Mike. The logic needed tweaking. When `OS.mac?` is `true` and `@version` is `nil`, the result should be `true`.